dashboard/app/helpers/aichat_helper.rb

```diff
@@ -19,7 +19,6 @@ def self.format_inputs_for_sagemaker_request(aichat_params, stored_messages, new
     inputs = BASE_PROMPT + aichat_params[:systemPrompt]
     inputs += aichat_params[:retrievalContexts].join(" ") if aichat_params[:retrievalContexts]
     inputs = SENTENCE_BEGIN_TOKEN + wrap_as_instructions(inputs)
-    # Filter only messages from user or assistant - ignore model update messages.
```

Comment:
Outside of scope of your PR, but I wonder if we should rename this file with something like `aichat_sagemaker_helper`.

Reply:
Related [discussion](https://github.com/code-dot-org/code-dot-org/pull/57944#discussion_r1561682568) from past PR.

Reply:
Sounds reasonable to me!